pkg/providers/nutanix/config/md-template.yaml

```diff
@@ -1,3 +1,105 @@
+{{- if .failureDomains }}{{- range $index, $fd := .failureDomains }}
```

Comment:
try adding `-` before `}}` to avoid adding unnecessary new lines here

Reply:
Fixed